.github/workflows/build.yml

```diff
@@ -64,6 +64,9 @@ jobs:
       with:
         persist-credentials: false
         submodules: recursive
-
```

Comment:
I'm not sure if these are really needed. Why isn't updating `.github/actions/kamel-prepare-env/action.yml` enough?

Reply:
I'll check that.